### PR TITLE
Add kusto extension

### PR DESCRIPTION
**Pull request by joberume**
Adding Kusto Extension entry on gallery

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -3537,6 +3537,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "77",
```
Comment (alanrenmsft) [inline extensionsGallery.json:3542]: you should simply copy the kusto extension entry ( you are using a different ID now.) in insider gallery and add it to the same order as insider gallery. makes it easier to compare the differences between insider and stable gallery files

**Review by alanrenmsft — CHANGES_REQUESTED** [review]
please see my comments

File: extensionsGallery.json (hunk 1 above)
Comment (joberume) [reply extensionsGallery.json:3542]: Got it, changing 👍

**Review by alanrenmsft — APPROVED** [review]

**Comment by alanrenmsft** [comment]
@joberume please work with @abist to make sure this goes into the RC gallery.

**Comment by Charles-Gagnon** [comment]
FYI - in the future if this is meant for the RC build it's best if you just use the single PR we have for updating the release gallery unless you specifically wanted to update and release the extension to stable immediately.

Generally we shouldn't be merging in changes to this file unless we're actually publishing the updated extension immediately since now we have a state where if someone needs to update the stable gallery then they'd have to revert this change first.

(ignore this if you are actually publishing the extension to stable now)

**Comment by alanrenmsft** [comment]
interesting and makes sense, is this the approach we decided recently? I never heard of this before

**Comment by Charles-Gagnon** [comment]
I did this with my latest run as release DRI but I guess I didn't follow through appropriately to make sure everyone was on the same page 🤦‍♂️

**Comment by abist** [comment]
I'll put this in the guide as I'm going through this process

**Comment by alanrenmsft** [comment]
this approach definitely makes sense to me, let's make sure the team knows about it, especially those who have merge permission on this branch.

**Pull request merged**